html-css/week3/index4.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>The signup form</title>
+    <link rel="stylesheet" href="./style.css" />
+  </head>
+  <body>
+    <header>
+      <div class="container header-container">
+        <div id="logo-box">
+          <img src="./images/clothes-logo.png" alt="logo" />
+        </div>
+        <nav>
+          <ul>
+            <li>Home</li>
+            <li>Collections</li>
+            <li>About us</li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+    <main>
+      <div class="container">
+        <section>
+          <h1>Sign up to yourcustomtailor.com</h1>
+        </section>
+        <section>
+          <form action="index4.html" method="get">
+            <h2>Step 4/4 - Payment</h2>
+            <div>
+              <h3>Progress</h3>
+              <div class="progress-box">
+                <div class="step4"></div>
+              </div>
+              <p></p>
```

Comment:
why empty `p`? Also, if you use empty tags they can be self closed like `<p />`

Reply:
The same